content/en/docs/reference/command-line-tools-reference/kubelet.md

```diff
@@ -16,16 +16,14 @@ through various mechanisms (primarily through the apiserver) and ensures that
 the containers described in those PodSpecs are running and healthy. The
 kubelet doesn't manage containers which were not created by Kubernetes.
 
-Other than from a PodSpec from the apiserver, there are three ways that a
+Other than from a PodSpec from the apiserver, there are two ways that a
 container manifest can be provided to the Kubelet.
 
 - File: Path passed as a flag on the command line. Files under this path will be
   monitored periodically for updates. The monitoring period is 20s by default
   and is configurable via a flag.
 - HTTP endpoint: HTTP endpoint passed as a parameter on the command line. This
   endpoint is checked every 20 seconds (also configurable with a flag).
-- HTTP server: The kubelet can also listen for HTTP and respond to a simple API
-  (underspec'd currently) to submit a new manifest.
```

Comment:
The above text comes from the upstream source code: https://github.com/kubernetes/kubernetes/blob/master/cmd/kubelet/app/server.go#L138-L156

The text also shows up when you do `kubelet --help`.

It is inaccurate. Since we were unable to generate reference docs for `kubelet` for over two years, this change is acceptable in my opinion.